From 99302b359e1ca1f2f4a70451517576ae1f9bc4aa Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Fri, 3 Jan 2025 13:40:17 -0800
Subject: [PATCH] Don't use git for cursive

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 494c642..f137987 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -7,7 +7,7 @@ edition = "2021"
 ansi_term = "0.12"
 clap = { version = "4.0", features = ["derive"] }
 csv = "1.3.0"
-cursive = { git = "https://github.com/gyscos/cursive", default-features = false, features = ["crossterm-backend"] }
+cursive = { version = "0.20", default-features = false, features = ["crossterm-backend"] }
 include_dir = { version = "0.7.3", features = ["glob"] }
 lazy_static = "1.4.0"
 nix = "0.26.2"
-- 
2.47.1

